README.md

```diff
@@ -128,11 +128,12 @@ Here are some of the things coming soon:
 
   - [x] Support `$PEBBLE_SOCKET` and default `$PEBBLE` to `/var/lib/pebble/default`
   - [x] Define and enforce convention for layer names
-  - [ ] Dynamic layer support over the API
+  - [x] Dynamic layer support over the API
+  - [x] Configuration retrieval commands to investigate current settings
```

Comment:
Note: dynamic layer support and the configuration/plan API was added in a previous PR.